frontend/src/Component/Share/Footer/Footer.jsx

```
@@ -0,0 +1,51 @@
+const Footer = () => {
+    return (
+        <div className="bg-gradient-to-r from-gray-900 to-gray-700 text-white py-10">
+            <footer className="footer footer-center p-10">
+                <nav className="flex flex-wrap justify-center gap-6 text-lg font-semibold">
+                    <a className="hover:text-gray-300 transition duration-300">About Us</a>
+                    <a className="hover:text-gray-300 transition duration-300">Contact</a>
+                    <a className="hover:text-gray-300 transition duration-300">Jobs</a>
+                    <a className="hover:text-gray-300 transition duration-300">Press Kit</a>
+                </nav>
+                <nav className="mt-6 flex gap-6 justify-center">
+                    <a className="p-3 rounded-full bg-gray-800 hover:bg-gray-600 transition duration-300">
```

Biome (1.9.4)

[error] Provide a href attribute for the a element.

An anchor element should always have a href
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)